c/apps/lf-workspace/hello/src/scripts/Commands.txt

```diff
@@ -0,0 +1,5 @@
+lfc c/apps/lf-workspace/hello/src/HelloWorld.lf
```

Comment:
These commands should be accompanied by desctitions about what they are intended to do.